### DISCUSSION TITLE
How to judge slot if it is empty

### DISCUSSION THREAD

**Discussion by FTYears** [PrismarineJS/mineflayer#2529] [Q&A] [answered]
I use the 'bot.setquickbarslot(0)' but I want to know if it is empty. How should I write ? Thanks for your reply.

**Comment by amoraschi** [comment, marked as answer]
Check it with bot.heldItem

**Reply by FTYears** [reply]
OK Thanks a lot.

**Reply by amoraschi** [reply]
Can you mark it as resolved? So it helps other people, thanks

**Reply by FTYears** [reply]
emmm sorry I 'm new. I donot know how to do it:(
